test/test_db.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, idx, base_port=32000):
         self.idx = idx
         self.name = f"Db{idx}"
         self.port = base_port + self.idx
-        self.seed_port = base_port + 100
+        self.seed_port = base_port + 2000
```

Comment:
This is just a temporary workaround until RPC ports (as opposed to gossip ports) are learned dynamically. There is a hardcoded offset of 2000 that I temporarily assumed for the workaround until this is implemented, and I changed this to match the testing script for it to pass, since I assumed the offset in the testing script was arbitrary. Will roll back this change and fix the workaround on my side.